src/api/migraphx.py

```diff
@@ -178,14 +178,67 @@ def shapes(h):
              returns='const migraphx::shape&')
 
 
+@auto_handle()
+def instruction_ref(h):
+    h.constructor('create')
+
+
+@api.handle('migraphx_instructions_refs',
+            'std::vector<migraphx::instruction_ref>')
+def instructions_refs(h):
+    h.constructor('create',
+                  api.params(ptr='migraphx_instruction_ref_t*', size='size_t'),
+                  fname='migraphx::to_obj_vector<migraphx_instruction_ref_t>')
+    h.method('size', returns='size_t')
+    h.method('get',
+             api.params(idx='size_t'),
+             fname='at',
+             cpp_name='operator[]',
+             returns='const migraphx::instruction_ref&')
+
+
+@api.handle('migraphx_modules_refs', 'std::vector<migraphx::module*>')
+def modules_refs(h):
```

Comment:
This should probably be called `modules` as the module class it already a `module_ref`.

Reply:
Changed it to just `modules`.